showcase/src/assets/examples/reactive-forms-with-ngx-form-errors/reactive-forms-with-ngx-form-errors.scss

```diff
@@ -0,0 +1,90 @@
+$monitor-query: "screen and (min-width:1200px)";
+$table-query: "screen and (max-width: 1200px)";
+$mobile-query: "screen and (max-width: 600px)";
```

Comment:
Can you change these 3 lines too to keep them in sync with the source file `welcome/pages/reactive-form-errors/demo-reactive-form-errors-page.component.scss`

In fact, can you update this file entirely?